deployments/otel-jaeger/docker-compose.yml

```diff
@@ -3,24 +3,13 @@ version: "2"
 services:
   # Jaeger
   jaeger-all-in-one:
-    image: jaegertracing/all-in-one:1.47
+    image: jaegertracing/all-in-one:latest
```

Comment:
I think it would be better to use a pinned version of the image, rather than `latest`, as `latest` will change over time, potentially introducing breaking changes. We're better bumping the version manually ourselves as this gives us control and improves overall stability.

Reply:
makes sense, changed!